samples/matter/common/src/task_executor.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "task_executor.h"
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+constexpr size_t kEventQueueSize = 10;
```

Comment:
I think it was ok to hardcode the queue size, as long, as we did it in the example app (app_task.cpp). However now, we try to create a generic module that will be recommended to use by the customers. I don't think it's a good idea to hardcode the size and potentially make the customer modify the common module in case 10 is not enough. It should be configurable via e.g Kconfig.